pkg/cli/cmds/server.go

```diff
@@ -208,20 +209,25 @@ var ServerFlags = []cli.Flag{
 	ClusterDomain,
 	cli.StringFlag{
 		Name:        "flannel-backend",
-		Usage:       "(networking) backend<=option1=val1,option2=val2> where backend is one of 'none', 'vxlan', 'ipsec', 'host-gw', 'wireguard-native', or 'wireguard' (deprecated)",
+		Usage:       "(deprecated) use --flannel-opts",
 		Destination: &ServerConfig.FlannelBackend,
 		Value:       "vxlan",
 	},
 	cli.BoolFlag{
 		Name:        "flannel-ipv6-masq",
-		Usage:       "(networking) Enable IPv6 masquerading for pod",
+		Usage:       "(deprecated) use --flannel-opts",
```

Comment:
We need to add `logrus.Warn` when these flags are used as per https://github.com/dereknola/k3s/blob/master/docs/adrs/deprecating-and-removing-flags.md